packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -263,15 +263,15 @@ test(testCase, async (t: Test) => {
       );
     } catch (e) {
       t2.equal(
-        e.response.status,
+        e?.response?.status,
         400,
         `Endpoint ${fInvoke} without required contractName: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e.response.data.map((param: { path: string }) =>
         param.path.replace(".body.", ""),
       );
       t2.ok(
-        fields.includes("contractName"),
+        fields?.includes("contractName"),
```

Comment:
@micoferdinand98 I think the question mark is not needed anymore, could you please double check?